dir.props

```diff
@@ -35,6 +35,7 @@
     <!-- list of nuget package sources passed to NuGet -->
     <RestoreSources>
       https://dotnet.myget.org/F/dotnet-core/api/v3/index.json;
+      https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json;
       https://dotnet.myget.org/F/cli-deps/api/v3/index.json;
```

Comment:
The `cli-deps` feed was deleted.  If you need to make anymore changes to this PR, can you also remove this?  But you don't have to reset CI just for this one change.